docs/docs/how-tos/nebari-destroy.md

```diff
@@ -0,0 +1,180 @@
+
+:::info
+Even with this script, AWS on occasion will find itself in some resource dependency conflict and you might need to destroy these resources manually from the AWS console as shown [above](#delete-resources-from-the-aws-console).
+:::
+
+
+### Azure
+
+If you deployed your Nebari cluster on Azure and the `nebari destroy` command failed, you will need to destroy one resources, the resource group.
+
+You will need [Azure's CLI `az`](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) installed and configured. Then run the following.
+
+```bash
+az group delete --resource-group "<project-name>-<namespace>"
+```
+
+Or if you'd prefer, you can destroy the resource group from the Azure portal, [follow these instructions](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#delete-resource-groups).
+
+### Digital Ocean
+
+If you deployed your Nebari cluster on Digital Ocean and the `nebari destroy` command failed, you will need to destroy two resources, the Kubernetes cluster (DOKS) and the S3 bucket (a Digital Ocean resource, not AWS).
+
+Digital Ocean offers an S3-compatible cloud object storage and requires the use of the [`aws` CLI](https://aws.amazon.com/cli/) to destroy it.
+
+```bash
+aws s3 rb s3://<project-name>-<namespace>-terraform-state --force --endpoint=https://<region>.digitaloceanspaces.com
+```
+
+Finally, you will need to destroy the Kubernetes cluster with [Digital Ocean's CLI tool, `doctl`](https://docs.digitalocean.com/reference/doctl/).
+
+```bash
+doctl kubernetes cluster delete <project-name>-<namespace> -f
+```
+
+Or, if you wish to destroy your Digital Ocean resources from the Digital Ocean control panel and follow these instructions to destroy your [Kubernetes cluster](https://docs.digitalocean.com/products/kubernetes/how-to/destroy-clusters/) and these instructions to destroy the associated [space (i.e. S3)](https://docs.digitalocean.com/products/spaces/how-to/destroy/). 
+
+
+### Google Cloud Platform (GCP)
+
+If you deployed your Nebari cluster on GCP and the `nebari destroy` command failed, you will need to destroy two resources, the Kubernetes cluster (GKE) and cloud storage bucket (GCS).
+
+You will need Google's [`gcloud`](https://cloud.google.com/sdk/gcloud) and [`gsutil`](https://cloud.google.com/storage/docs/gsutil).
+
+To destroy the cloud storage bucket, run the following.
+
+```bash
+gsutil -m rm -r gs://<project-name>-<namespace>-terraform-state
+```
+
+And to destroy the Kubernetes cluster, run the following.
+
+```bash
+gcloud container clusters delete <project-name>-<namespace> --region <region>
+```
+
+If you wish to destroy the cloud storage bucket from the GCP console, [follow these instructions](https://cloud.google.com/storage/docs/deleting-buckets). And to destroy the Kubernetes cluster from the GCP console, [follow these instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/deleting-a-cluster#console).
```

Comment:
Nitpick for accessibility, people using screen readers can benefit from explicit link texts :) 

```suggestion
If you wish to destroy the cloud storage bucket from the GCP console, [follow these instructions on Deleting buckets](https://cloud.google.com/storage/docs/deleting-buckets). And to destroy the Kubernetes cluster from the GCP console, [follow these instructions on Deleting a cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/deleting-a-cluster#console).
```